rst.rst

```diff
@@ -45,6 +51,8 @@ Lincoln said:
 Lists
 *****
 
+.. note: RST requires extra indentation for list items that span multiple lines or paragraphs, whereas MD does not.
```

Comment:
As a stylistic matter, I didn't include phrases like "whereas MD does not."  Perhaps the best thing here is to show an example of the indentation you need?

Reply:
Oh, this was mistaken, or at least not straightforward enough to include, I meant to drop it.